test/parallel/test-timers-next-tick.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const common = require('../common');
+
+// This test documents an internal implementation detail of the Timers:
+// since timers of different durations are stored in separate lists,
+// a nextTick queue will clear after each list of timers. While this
+// behaviour is not documented it could be relied on by Node's users.
```

Comment:
It was implied before - maybe it should be documented barring what I said earlier doesn't prevent the nextTick calls?

Reply:
Personally, I don't necessarily want to document it. We might want to have them run between individual timers in the future, honestly. That would be closer to spec & browsers.

Reply:
Hmm, fair enough.